src/components/shell/flyouts/profile/profile.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import { Trans } from 'react-i18next';
+
+import Config from 'app.config';
+import { svgs, getEnumTranslation } from 'utilities';
+import {
+  Btn,
+  BtnToolbar,
+  ErrorMsg,
+  Hyperlink,
+  PropertyGrid as Grid,
+  PropertyRow as Row,
+  PropertyCell as Cell,
+} from 'components/shared';
+import Flyout from 'components/shared/flyout';
+
+import './profile.css';
+
+const Section = Flyout.Section;
+
+export const Profile = (props) => {
+  const { t, user, logout, onClose } = props;
+
+  const roleArray = Array.from(user.roles);
+  const permissionArray = Array.from(user.permissions);
+
+  return (
+    <Flyout.Container>
+      <Flyout.Header>
+        <Flyout.Title>{t('profileFlyout.title')}</Flyout.Title>
+        <Flyout.CloseBtn onClick={onClose} />
+      </Flyout.Header>
+      <Flyout.Content className="profile-container">
+        {
+          !user &&
+          <div className="profile-container">
+            <ErrorMsg className="profile-error">{t("profileFlyout.noUser")}</ErrorMsg>
+            <Trans i18nKey={'profileFlyout.description'}>
+              <Hyperlink href={Config.contextHelpUrls.rolesAndPermissions} target="_blank">{t('profileFlyout.learnMore')}</Hyperlink> about roles and permisions
```

Comment:
At the end of this line there is some text, seems like a copy/paste error?

Reply:
Ahh looks like  the text is already covered in the translations so it can be removed?

Reply:
There has to be some sort of text there as it sets the template for what it gets replaced with.